app/views/users/registrations/new.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Sign up</h2>
+
+<%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+  <%= render "users/shared/error_messages", resource: resource %>
+
+  <div class="field">
+    <%= f.label :name %><br />
+    <%= f.text_field :name, autofocus: true, autocomplete: "name" %>
+  </div>
+
+  <div class="field">
+    <%= f.hidden_field :post_counter, value: 0 %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :email_confirmation %><br />
+    <%= f.email_field :email_confirmation, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %>
+    <% if @minimum_password_length %>
+    <em>(<%= @minimum_password_length %> characters minimum)</em>
```

Comment:
- [ ] I would recommend updating the UI for login and registration to match the wireframe and their styling.

### Your design 

### Sign-up page
![Screenshot from 2021-12-01 03-38-45](https://user-images.githubusercontent.com/22434660/144136275-5d53eef4-744e-4256-9615-daa41b22aa82.png)

### Sign-in page

![Screenshot from 2021-12-01 03-38-24](https://user-images.githubusercontent.com/22434660/144136505-abf00cfc-bd17-41a9-bf18-780053bbae84.png)


### Given design

![Screenshot from 2021-12-01 03-39-07](https://user-images.githubusercontent.com/22434660/144136365-6f72aa68-ecf9-4a05-a5dd-3e031ec441e9.png)